run_tests.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Searches for and runs all tests in this repository."""
+# Testing
+from absl.testing import absltest
+
+# Logging
+from absl import logging
+
+# We use these to import all tests
+import glob
+import subprocess
+import os
+import inspect
+from importlib import import_module
+
+# Folders which can have tests
+TEST_FOLDERS = ['./tests', './src', './evaluations']
```

Comment:
I wonder if it would be helpful to allow the TEST_FOLDERS to be specified via a command line flag?

Reply:
cee9068